lib/src/value/calculation.dart

```diff
@@ -3,10 +3,13 @@
 // https://opensource.org/licenses/MIT.
 
 import 'package:meta/meta.dart';
+import 'dart:math' as math;
```

Comment:
Nit: Imports should be divided into sections, with `dart:` imports first, then `package:` imports, then relative imports (with each section sorted alphabetically)

See https://dart.dev/effective-dart/style#do-place-dart-imports-before-other-imports

Reply:
Thanks, done!